models/provisioning.cattle.io.cluster.js

```diff
@@ -86,6 +86,13 @@ export default class ProvCluster extends SteveModel {
       enabled:    canSnapshot,
     });
 
+    insertAt(out, idx++, {
+      action:  'restoreSnapshotAction',
+      label:   'Restore Snapshot',
+      icon:    'icon icon-fw icon-backup-restore',
+      enabled: canSnapshot,
+    });
+
```

Comment:
Since we're in here, could we change the labels for the actions to be localized?